addons/mail/tests/common.py

```diff
@@ -1092,20 +1092,17 @@ def assertBusNotifications(self, channels, message_items=None, check_unique=True
             }, {...}]
         """
         bus_notifs = self.env['bus.bus'].sudo().search([('channel', 'in', [json_dump(channel) for channel in channels])])
-        if check_unique:
-            self.assertEqual(len(bus_notifs), len(channels))
```

Comment:
Note: having this check at the end allows for better debugging in case of issues (seeing which one are missing for example)